modules/text_server_adv/text_server_adv.cpp

```diff
@@ -408,13 +405,12 @@ bool TextServerAdvanced::load_support_data(const String &p_filename) {
 	if (!icu_data_loaded) {
 		String filename = (p_filename.is_empty()) ? String("res://") + _MKSTR(ICU_DATA_NAME) : p_filename;
 
-		Ref<File> f;
-		f.instantiate();
-		if (f->open(filename, File::READ) != OK) {
+		Ref<FileAccess> f = FileAccess::open(filename, FileAccess::READ);
+		if (f.is_null()) {
 			return false;
 		}
 		uint64_t len = f->get_length();
-		PackedByteArray icu_data = f->get_buffer(len);
+		PackedByteArray icu_data = f->get_buffer_bind(len);
```

Comment:
The only reason it was using `File`, instead of `FileAccess` is to be buildable as both GDExtension and module, so it won't work as GDExtension with `get_buffer_bind`.

For consistency with GDExtension, I would rename original to `_get_buffer` and use `get_buffer` instead of  `get_buffer_bind` for a new one. Same for `store_buffe`

Reply:
This have caused soo many changes ;_;